pkg/cri/annotations/annotations.go

```diff
@@ -32,6 +32,14 @@ const (
 	// SandboxID is the sandbox ID annotation
 	SandboxID = "io.kubernetes.cri.sandbox-id"
 
+	// SandboxCPU is the initial CPU configuration for this sandbox
+	SandboxCPUPeriod = "io.kubernetes.cri.sandbox-cpu-period"
+	SandboxCPUQuota  = "io.kubernetes.cri.sandbox-cpu-quota"
+	SandboxCPUShares = "io.kubernetes.cri.sandbox-cpu-shares"
+
+	// SandboxMemory is the initial amount of memory associated with this sandbox
+	SandboxMem = "io.kubernetes.cri.sandbox-memory"
+
```

Comment:
nit .. probably want to explain that these represent "the sum of container resources for this sandbox" and does not include pod overhead?

Reply:
Updated. PTAL.